MaterialComponents.podspec

```diff
@@ -91,6 +91,16 @@ Pod::Spec.new do |s|
     end
   end
 
+  s.subspec "BottomAppBar" do |ss|
+    ss.ios.deployment_target = '8.0'
+    ss.public_header_files = "components/#{ss.base_name}/src/*.h"
+    ss.source_files = "components/#{ss.base_name}/src/*.{h,m}", "components/#{ss.base_name}/src/private/*.{h,m}"
+
+    ss.dependency "MaterialComponents/Buttons/Component"
```

Comment:
Do we need "/Component"? I thought that was only for the themers.

Reply:
Done.